### DISCUSSION TITLE
Is it possible to use DrawingView or SignaturePad on MAUI Blazor hybrid?

### DISCUSSION THREAD

**Discussion by Dsalazar09** [CommunityToolkit/Maui#1806] [General]
Hello community, I require a space to sign on a Maui Blazor hybrid project, but it has not been possible to find a solution. Could you help me find a solution to the problem I have?

**Comment by CliffAgius** [comment]
You need to use a Blazor component rather than a MAUI component as all MAUI is doing is providing a Web component to run your Blazor code.

I have used this one in the past and it worked well for me [Blazorise SignaturePad](https://blazorise.com/docs/extensions/signaturepad)

Hope that helps.
